torch/_dynamo/variables/dicts.py

```diff
@@ -417,6 +417,8 @@ def call_method(
         elif name in ("pop", "get") and len(args) in (1, 2) and args[0] not in self:
             # missing item, return the default value. Install no DICT_CONTAINS guard.
             self.install_dict_contains_guard(tx, args)
+            if name == "pop":
+                raise_observed_exception(KeyError, tx)
```

Comment:
Looks like old test failed -- why would we want to raise here?

Reply:
thats python behavior ... KeyError if len(args) == 1 .. updated just now